src/patch.js

```diff
@@ -54,20 +49,22 @@ function createElement(node, isSVG) {
 function setElementProp(element, name, value, oldValue) {
   if (name === "key") {
   } else if (name === "style") {
-    for (var name in merge(oldValue, (value = value || {}))) {
-      element.style[name] = value[name] != null ? value[name] : ""
+    for (var i in merge(oldValue, (value = value || {}))) {
+      element.style[i] = value[i] != null ? value[i] : ""
```

Comment:
@mindplay-dk 
```js
element[name][i]
```

Reply:
Why would you want to do that dynamically? We know it'll only work with `style`, and it's an isolated use-case.

Reply:
It saves a few bytes.

Reply:
2 to be exact 😆